src/main/java/strategy/ChooserStrategy.java

```diff
@@ -0,0 +1,16 @@
+package strategy;
+
+import java.util.Map;
+
+public class ChooserStrategy {
```

Comment:
let's rename it. Strategy it's about choosing the right option, so we are getting line 'buttery butter' aka 'масло масляне')
It can be OperationStrategy